Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -390,10 +390,12 @@ void SeedFinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
         }
 
         // use geometric average
-        float cotThetaAvg2 = cotThetaB * cotThetaT;
+        float cotThetaAvg2;
         if (m_config.arithmeticAverageCotTheta) {
           // use arithmetic average
           cotThetaAvg2 = std::pow((cotThetaB + cotThetaT) / 2, 2);
+        } else if (cotThetaAvg2 = cotThetaB * cotThetaT; cotThetaAvg2 <= 0) {
+          continue;
```

Comment:
in this case I would make the calculation explicit like above and then check for the error case even tho it looks much more verbose

```suggestion
        } else {
          cotThetaAvg2 = cotThetaB * cotThetaT;
          if (cotThetaAvg2 <= 0) {
            continue;
          }
```

Reply:
and also move the comment `// use geometric average` down

Reply:
Reverting this as the simplest way to fix the clang-tidy warning about the uninit variable.